### ISSUE TITLE
SPIKE: Finalize Database Schema

### ISSUE THREAD

**Issue by mackcooper1408** [sfbrigade/datasci-earthquake#29]
## Context
Before we can begin actually defining models in the code and running migrations, we need to know _exactly_ what tables and fields we need

## Definition of Done
1. An initial database schema is thoroughly documented and laid out for us to use
2. Schema is approved and placed in team documentation

## Engineering Details
this does not need to be perfect as data will always evolve, but should be a good foundation of the minimum necessary tables and fields to get the app up and running

**Comment by oscarsyu** [comment]
should this be consolidated with the data model ticket I put?

**Comment by mackcooper1408** [comment]
yeah, makes sense, I'll mark this as a duplicate of #11
